Example/TrustlySDK/ViewController.swift

```diff
@@ -67,9 +68,24 @@ class ViewController: BaseViewController {
     // MARK: - IBActions
     
     @IBAction func pay(_ sender: Any) {
+
+        if let amountText = amountTextView.text,
+           let amount = Double(amountText) {
+
+            establishData["amount"] = String(format: "%.2f", amount)
+        }
+
+        if let emailText = emailTextView.text, !emailText.isEmpty {
+
+            establishData["customer.email"] = emailText
+
+        }
+
+        self.openLightbox()
         
-        let trustlyLightboxViewController = TrustlyLightBoxViewController()
-        trustlyLightboxViewController.delegate = self
+    }
+    
+    @IBAction func payWituAESWebAuth(_ sender: Any) {
```

Comment:
Maybe there's a typo here in the word `with`?

Reply:
Fixed